src/dotnet/commands/dotnet-cache/Program.cs

```diff
@@ -71,10 +74,43 @@ public static int Run(string[] args)
                 cache.ExtraMSBuildArguments = app.RemainingArguments;
                 cache.ProjectArgument = projectArgument.Value();
                
-                return cache.Execute();
+                return 0;
             });
 
-            return app.Execute(args);
+            int exitCode = app.Execute(args);
```

Comment:
I think it would be better to solve this on different occasion - this PR already has grew enough and it is not clear to me yet how to do it without adding more code than it would reduce